### PR TITLE
Globalize Loopback-boot

### PR DESCRIPTION
**Pull request by davidcheung**
@0candy PTAL

### CODE REVIEW

**Comment by davidcheung** [comment]
@slnode test please

File: lib/executor.js [hunk 1]
```diff
@@ -230,12 +232,12 @@ function defineModels(app, instructions) {
     if (!data.definition) {
       model = registry.getModel(name);
       if (!model) {
-        throw new Error('Cannot configure unknown model ' + name);
+        throw new Error(f.g('Cannot configure unknown model %s', name));
```
Comment (0candy) [inline lib/executor.js]: `g.f` not `f.g`

**Comment by 0candy** [comment]
Just 1 typo, otherwise LGTM.

**Comment by davidcheung** [comment]
@slnode test please

**Comment by davidcheung** [comment]
@slnode test please

**Pull request merged**